core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -188,6 +188,24 @@ class BrokerTopicMetrics(name: Option[String]) extends KafkaMetricsGroup {
     }
   }
 
+  /**
+   * Greedily initialize all topic metrics
+   */
+  def initializeMetrics(): Unit = {
+    messagesInRate
+    bytesInRate
+    bytesOutRate
+    bytesRejectedRate
+    replicationBytesInRate
+    replicationBytesOutRate
+    failedProduceRequestRate
+    failedFetchRequestRate
+    totalProduceRequestRate
+    totalFetchRequestRate
+    fetchMessageConversionsRate
+    produceMessageConversionsRate
```

Comment:
This is brittle. How do we ensure we don't forget to add an item here when we add a new `def`? We could either add a test to help us or change the code to make this kind of thing impossible. One way would be to replace `def` with `val` and have the laziness be optional within a class that wraps the `getAndMaybePut` logic.